clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -1280,10 +1285,10 @@ def session_to_viscode(session_name):
     Returns:
         M00 if is the baseline session or the original session name capitalized
     """
-    if session_name == "M00":
+    if session_name == "M000":
```

Comment:
The docstring is incorrect (I suspect a copy paste with the previous function).
Could you fix it and update for your changes at the same time ?